src/encoding.rs

```diff
@@ -29,6 +29,16 @@ impl EncodingKey {
 
     /// If you are loading a RSA key from a .pem file.
     /// This errors if the key is not a valid RSA key.
+    ///
+    /// # NOTE
+    ///
+    /// According to the [ring doc](https://briansmith.org/rustdoc/ring/signature/struct.RsaKeyPair.html#method.from_pkcs8),
+    /// the key should be at least 2047 bits.
+    ///
+    /// You can generate a key with the following:
+    ///
+    ///     openssl genrsa -out rsa-private.pem 2048
```

Comment:
I wouldn't put the openssl commands in the docs, it's already mentionned in the ring docs